recipes-extended/libcobalt/libcobalt_git.bb

```diff
@@ -51,10 +58,16 @@ do_install() {
         rm -rf ${STAGING_DIR_TARGET}/usr/lib/libcobalt.so
     fi
     install -d ${D}${libdir}
-    install -m 0755 ${S}/src/out/${COBALT_PLATFORM}_${COBALT_BUILD_TYPE}/lib/libcobalt.so ${D}${libdir}
+    install -m 0755 ${S}/src/out/${COBALT_PLATFORM_NAME}_${COBALT_BUILD_TYPE}/lib/libcobalt.so ${D}${libdir}
 
     install -d ${D}/usr/share
-    cp -prf ${S}/src/out/${COBALT_PLATFORM}_${COBALT_BUILD_TYPE}/content ${D}/usr/share/
+    cp -prf ${S}/src/out/${COBALT_PLATFORM_NAME}_${COBALT_BUILD_TYPE}/content ${D}/usr/share/
+
+    install -d ${D}/${includedir}/third_party/starboard/wpe/${COBALT_PLATFORM}/${COBALT_ARCH}
+    cp -prf ${S}/src/third_party/starboard/wpe/${COBALT_PLATFORM}/${COBALT_ARCH}/*.h ${D}/${includedir}/third_party/starboard/wpe/${COBALT_PLATFORM}/${COBALT_ARCH}/
+
+    install -d ${D}/${includedir}/third_party/starboard/wpe/shared
+    cp -prf ${S}/src/third_party/starboard/wpe/shared/*.h ${D}/${includedir}/third_party/starboard/wpe/shared/
 }
 
 COBALT_PACKAGE = " \
```

Comment:
${includedir}/* needs to be added to COBALT_PACKAGE so that the headers can be packaged by the FILES directive.